docs/source/example-c.rst

```diff
@@ -9,79 +9,79 @@ To simplify package management in the example,
 we'll use :term:`conda`.
 We'll also use :term:`git` to obtain the model source code.
 
-This is a somewhat long example.
-To break it up,
-here are the steps we'll take:
+Here are the steps we'll take to complete this example:
 
 #. Create a :term:`conda environment` that includes software to compile the
    model and wrap it with the *babelizer*
 #. Clone the `bmi-example-c`_ repository from GitHub and build the
    *heat* model from source
 #. Create a *babelizer* input file describing the *heat* model
-#. Run the *babelizer* to generate Python bindings, then build the bindings
+#. Run the *babelizer* to generate a Python package, then build and install the package
 #. Show the *heat* model running in Python through *pymt*
 
 Before we begin,
 create a directory to hold our work:
 
 .. code:: bash
 
-  $ mkdir build && cd build
+  mkdir example-c && cd example-c
 
 This directory is a starting point;
 we'll make new directories under it as we proceed through the example.
 In the end,
-the directory structure under ``build`` should look like that in Figure 1.
+the directory structure under ``example-c`` should look like that in Figure 1.
 
 .. figure:: _static/babelizer-bmi-example-c.png
```

Comment:
Instead of using a png for this, how about using the output of a ``tree`` command? Apart from looking better (in my opinion), it would be easier to keep up-to-date and would be one less binary file in version control.

Reply:
I'm thinking about removing it entirely because it's hard to maintain. It was a request from a reviewer on the JOSS paper.

Reply:
Changed in fa14f96.